src/mjolnir/graphvalidator.cc

```diff
@@ -630,8 +630,7 @@ void GraphValidator::Validate(const boost::property_tree::ptree& pt) {
       }
       sum += density;
     }
-    float average_density = sum / densities[level].size();
```

Comment:
its usage is under LOG_DEBUG macro so I just moved whole expression

Reply:
if somebody is in love with variable we can do it the other way https://github.com/valhalla/valhalla/pull/2922#discussion_r591666498